backend/pgnService/game/chesscom.ts

```diff
@@ -0,0 +1,148 @@
+import tcn from '@savi2w/chess-tcn';
+import axios from 'axios';
+import { Chess } from 'chess.js';
+import { Browser, BrowserErrorCaptureEnum } from 'happy-dom';
```

Comment:
happy-dom isn't added to package.json, so this causes sls to fail when trying to deploy